apps/ai-image-generator/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@contentful/ai-image-generator",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@contentful/app-sdk": "4.22.0",
+    "@contentful/f36-components": "4.45.0",
+    "@contentful/f36-tokens": "4.0.2",
+    "@contentful/react-apps-toolkit": "1.2.16",
+    "contentful-management": "10.38.3",
+    "emotion": "10.0.27",
```

Comment:
We ran into issues with emotion and are using `"@emotion/react": "^11.11.1",` in AICG